_posts/2024-12-05-this-month-in-servo.md

```diff
@@ -19,6 +19,20 @@ Last month we also landed support for **‘justify-self’** on positioned eleme
 - **“AES-KW”** algorithm in **generateKey()**, **importKey()**, **wrapKey()**, and **unwrapKey()** (#34262)
 - **“PBKDF2”** (#34164) and **“HKDF”** (#34200) algorithms in **deriveBits()**
 
+## More engine changes
+
+Layout performance continues to improve, with faster layout for **‘column’ flex containers** (@Loirooriol, @mrobinson, #34346) and faster caching of intrinsic sizes (@Loirooriol, #34384), in addition to parallel layout for flexbox (@mrobinson, #34132).
+
+We’ve fixed the behaviour of **offsetLeft** and **offsetTop** relative to &lt;body> with ‘position: static’ (@nicoburns, @Loirooriol, #32761), which also required spec changes (@nicoburns, @Loirooriol, csswg-drafts#10549).
+We’ve also fixed several layout bugs around:
+
+- **‘object-fit’** when ‘aspect-ratio’ is set (@Loirooriol, #34413)
+- **clientWidth** and friends on floats and replaced inlines ()
```

Comment:
Missing author and PR.

Reply:
fixed in 6028773e154c465255507917fc753fa623a980ec

Reply:
fixed in dd61940d3b8133a1422522bea717e8612f5228b7